src/elements/common/feature-checking/util.ts

```diff
@@ -0,0 +1,14 @@
+import * as types from './flowTypes';
+// Use require instead of import to avoid TypeScript error
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const get = require('lodash/get');
```

Comment:
You shouldn't need to do this. Please revert the code to what is in the flow file.